src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -175,6 +175,11 @@ public BrowserTest(int secondsBeforeTimeout) {
         ensureActiveTabIsNotClosed();
     }
 
+    public BrowserTest(String appium){
+
```

Comment:
Agree, but somehow the timeout setting was called even though I had it overridden in the AndroidAppTest class.